examples/rl/main.py

```diff
@@ -14,6 +15,8 @@
 NUM_STEPS = None
 CHECKPOINT_PATH = join(dirname(SCENARIO_PATH), "checkpoints")
 CHECKPOINT_INTERVAL = 5
+LOAD_PATH = join(dirname(SCENARIO_PATH), "checkpoints_backup")
```

Comment:
could it be configurable? (like it could be same as "checkpoints")

Reply:
in config/templates.yml, there does have "load_path" and "load_episode",  and also some other configs